pennylane/ops/op_math/controlled_class.py

```diff
@@ -305,6 +306,20 @@ def pow(self, z):
             for op in base_pow
         ]
 
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + self.base.arithmetic_depth
+
+    def simplify(self, depth=-1) -> "ControlledOp":
+        if depth == 0:
+            return self
+        return ControlledOp(
```

Comment:
```suggestion
        if isinstance(self.base, Controlled):
            base = self.base.base.simplify(depth=0 if depth==1 else depth-2)
            return Controlled(base,
                control_wires = self.control_wires + self.base.control_wires,
                control_values = self.control_values + self.base.control_values,
                work_wires = self.work_wires + self.base.work_wires
            )
        return ControlledOp(
```
This would cover:
```
op1 = Controlled(qml.PauliX(0), 1)
op2 = Controlled(op1, 2)
op2.simplify()
```

Reply:
Nice catch! Added.